File: src/targets/csharp/restsharp/fixtures/jsonObj-multiline-linebreaks.cs
var client = new RestClient("https://httpbin.org/anything");
var request = new RestRequest(Method.POST);
request.AddHeader("content-type", "application/json");
request.AddParameter("application/json", "{\"message\":\"foo\\\\nbar\\\\nbaz\"}", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);

File: src/targets/javascript/fetch/fixtures/jsonObj-multiline-linebreaks.js
const options = {
  method: 'POST',
  headers: {'content-type': 'application/json'},
  body: JSON.stringify({message: 'foo\nbar\nbaz'})
};

fetch('https://httpbin.org/anything', options)
  .then(response => response.json())
  .then(response => console.log(response))
  .catch(err => console.error(err));